closure/goog/dom/browserrange/abstractrange.js

```diff
@@ -257,7 +257,7 @@ goog.dom.browserrange.AbstractRange.prototype.getHtmlFragment = function() {
           output.append(html);
         } else {
           var index = html.lastIndexOf('<');
-          output.append(index ? html.substr(0, index) : html);
+          output.append(index !== -1 ? html.slice(0, index) : html);
```

Comment:
This change isn't right, and is what is causing the browserrange tests to fail. The original logic is intentional:
 - if index is -1, then this appends empty string.
 - if index is 0, then the entire HTML content should be added
 - if the index is > 0, then only the portion of the html after the last open tag is appended.
 
 Can you add a comment to help document this?

Reply:
Good catch. I haven't seen it being done like this yet. I remember being confused about the `index ?` part when I changed it but I didn't think deeper about it.
I have added a comment about the wanted behaviour and also fixed the code to reflect that.

Reply:
Thanks! This LGTM.